### DISCUSSION TITLE
v0.1.9a4

### DISCUSSION THREAD

**Discussion by Luosuu** [ByteDance-Seed/VeOmni#724] [Announcements]
## What's Changed
* [model] fix: qwen3_moe router double-softmax and idempotent _init_weights wrap by @TimYangst in https://github.com/ByteDance-Seed/VeOmni/pull/715
* [ops, model] feat: dispatch Qwen3.5 linear-attention kernels via OpSlot by @TimYangst in https://github.com/ByteDance-Seed/VeOmni/pull/714
* [ops, model] feat: forward temperature and return entropy from log-probs path by @Luosuu in https://github.com/ByteDance-Seed/VeOmni/pull/720
* [ci, model] test: add bitwise logits-equal tests for transformers v4 models by @TimYangst in https://github.com/ByteDance-Seed/VeOmni/pull/721
* [release] chore: release v0.1.9a4 by @Luosuu in https://github.com/ByteDance-Seed/VeOmni/pull/723

**Full Changelog**: https://github.com/ByteDance-Seed/VeOmni/compare/v0.1.9a3...v0.1.9a4

<hr /><em>This discussion was created from the release <a href='https://github.com/ByteDance-Seed/VeOmni/releases/tag/v0.1.9a4'>v0.1.9a4</a>.</em>
